.circleci/config.yml

```diff
@@ -31,16 +39,18 @@ jobs:
 
       # save cocoapods version gem data
       - save_cache:
+          name: Saving Gemfile Cache
           key: 1-gems-{{ checksum "Gemfile.lock" }}
           paths:
             - vendor/bundle
 
       # restore pods related caches
       - restore_cache:
+          name: Restoring CocoaPods Cache
           keys:
-            - cocoapods-cache-v5-{{ arch }}-{{ .Branch }}-{{ checksum "Podfile.lock" }}
-            - cocoapods-cache-v5-{{ arch }}-{{ .Branch }}
-            - cocoapods-cache-v5
+            - cocoapods-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Podfile.lock" }}
```

Comment:
If you need to rotate the cache I'd suggest rotating up to "v6". I'd suspect CircleCI deletes any caches after 30 days, so the v1 cache is probably gone, but safer to rotate up to a new number.

Reply:
I don't need to. I will revert the change to v5